webapi/admin.go

```diff
@@ -30,7 +30,7 @@ type WalletStatus struct {
 type searchResult struct {
 	Hash           string
 	Found          bool
-	Ticket         database.Ticket
+	Ticket         *database.Ticket
```

Comment:
Why the change to use a pointer here?

Reply:
Whilst doing the work, there was a good reason to change this. However I think I made some later changes which made it unnecessary.

I have undone the change, no longer using pointers.